### DISCUSSION TITLE
Interactive Replit Landing Page

### DISCUSSION THREAD

**Discussion by IronCodem** [ReplAPI-it/ReplAPI.it-NodeJS#55] [Project Showcase]
I made a landing page on Replit for the project. [Link to spotlight page](https://replit.com/@YashasShah/ReplAPI-it?v=1)!

**Comment by rayhanadev** [comment]
Woah, that's pretty kewl dude!

Actually, if you can polish that CSS a bit, match up the colors with our theme, and maybe even use the package on the website I might go ahead and adopt that as our landing page xD.

Also @BD103 is also a major member of this team (and go ahead and include @EpicGamer007 since they're also working on something big for us)

**Reply by IronCodem** [reply]
Thanks! Mind if I share the project with you so you can do your chosen edits?
Yeah, I'll add them as well.

**Reply by abhay7dev** [reply]
Yeah, I agree with Ray on this. Changing up the colors to match our current scheme seems like a good idea.

**Reply by rayhanadev** [reply]
> Thanks! Mind if I share the project with you so you can do your chosen edits?
> Yeah, I'll add them as well.

Sure thing, however I'm working on updating the project's documentation right now 😴 (booorrriinnng).

**Reply by IronCodem** [reply]
>Changing up the colors to match our current scheme seems like a good idea.

Okay, I'll try, but can you ping me the colour scheme @EpicGamer007? I can implement them from there.

**Reply by IronCodem** [reply]
Or @RayhanADev can just edit it from there...?

**Reply by rayhanadev** [reply]
@RayhanADev sure, let me wrap up a few things

**Reply by IronCodem** [reply]
Is the colour scheme just the colours on the favicon?

**Reply by rayhanadev** [reply]
Yep!

**Reply by IronCodem** [reply]
Oh, bc I got the colours from he favicon. Could you tell me where each colour goes (blue - bg etc.)

**Reply by rayhanadev** [reply]
Same as the favicon, blue should be background and greens and pink are accents :).
